refresh.py

```
@@ -63,31 +61,21 @@ def validate_config(cfg):
-            "[WARN] Backup password is very short; consider using a longer password."
-        )
 
 
 # ----------------- Commands -----------------
 def run_command(cmd, capture=False):
     if CONFIG.get("dry_run"):
-        console.print(f"[dry-run] {cmd}")
-        return "" if not capture else None
+        console.print(f"[DRY-RUN] {cmd}")
+        return None
     result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)